packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionModule.tests.js

```diff
@@ -142,7 +142,9 @@ QUnit.module('Mention format', () => {
         assert.strictEqual(element.dataset.marker, '@', 'correct marker');
         assert.strictEqual(element.dataset.mentionValue, 'John Smith', 'correct value');
         assert.strictEqual(element.dataset.id, 'JohnSm', 'correct id');
-        assert.strictEqual(element.innerText, '@John Smith', 'correct inner text');
+
+        new MentionFormat({}, element);
```

Comment:
Won't it work if we only call this usual constructor instead of static `create` method above?